client/components/public/imports/SamplePuzzles.jsx

```diff
@@ -118,7 +124,9 @@ export default class SamplePuzzles extends Component {
           <Grid padded centered textAlign="left" stackable>
             <Grid.Row>
               <Grid.Column width={16}>
-                Below you can find some of the past Puzzles. Download them and try to solve them for yourself!
+                <Header as='h2'>Apology</Header>
+                <p className='h4'>The puzzles were more challenging than intended this year. Like all of you, I have been isolated for over a year. Working in a vacuum, I missed the mark on some hints and the amount of time needed in a virtual environment. The goal was to provide a fun, free, safe opportunity for anyone to play and puzzle for the day. I hope that you might still enjoy playing with the puzzles after the event.</p>
```

Comment:
I believe you're referencing the spans around lines 146-148, yeah. Spans are generally just nice because they don't have any inherent padding like paragraphs or headers. I'm just using the h4 className to match the styling of the text from other pages on the site.